### DISCUSSION TITLE
Why does the agent swing back and forth when it reaches the target?

### DISCUSSION THREAD

**Discussion by ytipocmk631** [recastnavigation/recastnavigation#726] [General]
https://github.com/user-attachments/assets/b94e7097-5c58-4127-bcd9-43a9396f7323

As shown in the video, the agent jitters back and forth when it arrives at the agent, how can I avoid this problem?

**Comment by rob-myers** [comment]
Maybe you need to change the agent's acceleration?
A larger max acceleration might solve this.
